uwrt_mars_rover_arm/uwrt_mars_rover_arm_hw/include/uwrt_mars_rover_arm_hw/visibility.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Remove this since it's not copying their header, just basing off the gcc wiki entry. 

Also fyi our repo is MIT licensed